<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidgetWordlist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Wordlist</string>
     </attribute>
     <widget class="QTextEdit" name="textEditWordlist">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>87</y>
        <width>256</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QWebView" name="webViewWordList" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>451</width>
        <height>371</height>
       </rect>
      </property>
      <property name="url" stdset="0">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCollocation">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>21</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtTop</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type a word or expresion and then press Collect Examples. You can drag and drop sentences to your wordlist.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLoad">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>54</y>
        <width>145</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Collect Examples...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPlaySentences">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen to Sentences</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../usr/share/icons/gnome/32x32/actions/gtk-media-play-ltr.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/actions/gtk-media-play-ltr.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Corpus</string>
     </attribute>
     <widget class="QTextEdit" name="textEditCorpus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Load File ...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../../usr/share/icons/gnome/32x32/actions/add.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/actions/add.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionExport_Wordlist"/>
    <addaction name="actionExport_Selected_Wordlist"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionExport_Wordlist">
   <property name="text">
    <string>Export Wordlist</string>
   </property>
  </action>
  <action name="actionExport_Selected_Wordlist">
   <property name="text">
    <string>Export Selected Wordlist</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
   <slots>
    <slot>playPause()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPlaySentences</sender>
   <signal>clicked()</signal>
   <receiver>webViewWordList</receiver>
   <slot>playPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>open()</slot>
  <slot>addFile()</slot>
 </slots>
</ui>
